src/hotspot/share/memory/metaspace.cpp

```diff
@@ -785,6 +785,11 @@ void Metaspace::global_initialize() {
       if (rs.is_reserved()) {
         log_info(metaspace)("Successfully forced class space address to " PTR_FORMAT, p2i(base));
       } else {
+        LogTarget(Debug, metaspace) lt;
+        if (lt.is_enabled()) {
+          LogStream ls(lt);
+          os::print_memory_mappings((char*)base, size, &ls);
```

Comment:
I saw the above log message as follows:
```
java -XX:+UnlockDiagnosticVMOptions -XX:CompressedClassSpaceBaseAddress=0xa0000000 -XX:CompressedClassSpaceSize=1g -Xlog:metaspace=debug -Xlog:os+map=trace -Xshare:off -version
[0.001s][info][metaspace]  - commit_granule_bytes: 65536.
[0.001s][info][metaspace]  - commit_granule_words: 8192.
[0.001s][info][metaspace]  - virtual_space_node_default_size: 8388608.
[0.001s][info][metaspace]  - enlarge_chunks_in_place: 1.
[0.001s][info][metaspace]  - use_allocation_guard: 0.
[0.036s][debug][metaspace] Range [a0000000-e0000000) contains: 
[0.036s][debug][metaspace] a0000000-120000000 rw-p 00000000 00:00 0 
[0.036s][debug][metaspace] 
Error occurred during initialization of VM
CompressedClassSpaceBaseAddress=0x00000000a0000000 given, but reserving class space failed.
```
Is it easy to get rid of the blank line before the line begins with "Error occurred..."?

Reply:
Okay.